README.md

```diff
@@ -45,18 +45,35 @@ the serializer generator:
 $ rails g serializer post
 ```
 
-### Support for PORO's and other ORM's.
```

Comment:
Hi @feministy I'm curious if you have a reference for this grammatical change or if it's just what you're used to.   I prefer the current state, myself, but I believe there's no consensus, right?